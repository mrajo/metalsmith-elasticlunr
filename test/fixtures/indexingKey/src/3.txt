---
title: Three
search: false
---
This page won't be indexed.