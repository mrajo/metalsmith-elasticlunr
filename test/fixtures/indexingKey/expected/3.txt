This page won't be indexed.